resources/testharness-shadowrealm-outer.js

```diff
@@ -5,6 +5,9 @@
  * Convenience function for evaluating some async code in the ShadowRealm and
  * waiting for the result.
  *
+ * In case of error, this function intentionally leaks the stack trace (if it is
```

Comment:
"leaks" seems like an odd turn of phrase here

Reply:
I meant "information leak" and didn't consider it could also be interpreted as "memory leak". I'll go with "exposes" instead.